test/src/unit-capi-capnp.cc

```diff
@@ -289,6 +289,8 @@ void ArraySchemaCapnp::remove_temp_dir(const std::string& path) {
     REQUIRE(tiledb_vfs_remove_dir(ctx_, vfs_, path.c_str()) == TILEDB_OK);
 }
 
+#ifdef TILEDB_SERIALIZATION
```

Comment:
Should we just remove this unit test? I think it is superceeded by the `unit-capi-array_schema` and `unit-capi-dense` test?